charts/puris/templates/backend-deployment.yaml

```diff
@@ -46,23 +46,35 @@ spec:
             - name: DATASOURCE_USERNAME
               value: "{{ .Values.postgresql.auth.username }}"
             - name: DATASOURCE_PASSWORD
-              value: "{{ .Values.postgresql.auth.password }}"
+              valueFrom:
+                secretKeyRef:
+                  name: "{{ .Values.postgresql.auth.existingSecret }}"
+                  key: "postgres-password"
```

Comment:
hi @tom-rm-meyer-ISST I tested it locally and get the same result as workflow run, backend fails connection to postgres.
I think this is the reason, key should refer to "password".